src/impl_/pyclass.rs

```diff
@@ -198,6 +198,80 @@ macro_rules! slot_fragment_trait {
     }
 }
 
+slot_fragment_trait! {
+    PyClass__getattribute__SlotFragment,
+
+    /// # Safety: _slf and _attr must be valid non-null Python objects
+    #[inline]
+    unsafe fn __getattribute__(
+        self,
+        py: Python,
+        slf: *mut ffi::PyObject,
+        attr: *mut ffi::PyObject,
+    ) -> PyResult<*mut ffi::PyObject> {
+        let res = ffi::PyObject_GenericGetAttr(slf, attr);
+        if res.is_null() { Err(PyErr::fetch(py)) }
```

Comment:
I'm not sure about this implementation, does it look correct?

Reply:
Looks good to me!